src/js/components/Search.js

```diff
@@ -163,7 +163,10 @@ export default class Search extends Component {
   }
 
   _onInputKeyDown (event) {
```

Comment:
I think that won't work because the _onEnter listener is only active when dropActive  is true

```
const activeKeyboardHandlers = {
      ...
      enter: this._onEnter,
      ...
    };
if (dropActive && ! prevState.dropActive) {
     ...
      KeyboardAccelerators.startListeningToKeyboard(this,
        activeKeyboardHandlers);
```